index.html

```diff
@@ -142,13 +142,13 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.8.
                 <li>change of date format in sign of life string now applies only to PC Engines</li>
                 <li>use CONFIG_LOCALVERSION as a coreboot version string</li>
                 <li>double sign-of-life during power on</li>
-                <li>Enabled IOMMU</li>
+                <li>enabled IOMMU</li>
             </ol>
             <li>Known issues:</li>
             <ol>
-                <li>some PCIe cards are not detected on certain OSes</li>
-                <li>booting with 2 USB 3.x sticks plugged in apu4 sometimes results in detecting only 1 stick</li>
-                <li>certain USB 3.x sticks happen to not appear in boot menu</li>
+                <li><a href="pcengines/apu2-documentation#115">some PCIe cards are not detected on certain OSes</a></li>
```

Comment:
@mscibisz these are not links. They will not redirect to desired issue. Please use `Copy link address` option, because github interprets its own issues and creates fancy hyperlink like `pcengines/apu2-documentation#115`. Please fix for all links